bitseq/store.go

```diff
@@ -70,46 +70,47 @@ func (h *Handle) Exists() bool {
 	return h.dbExists
 }
 
+// New method returns a handle based on the receiver handle
+func (h *Handle) New() datastore.KVObject {
+	h.Lock()
+	defer h.Unlock()
+
+	return &Handle{
+		app:   h.app,
+		id:    h.id,
+		store: h.store,
+	}
+}
+
+// Copy deep copies the handle into the passed destination object
+func (h *Handle) Copy(o datastore.KVObject) error {
```

Comment:
Well, the problem is `MetaDataClone` is not to my personal taste :-). So I've decided to come to a compromise on this "baby naming" conflict. I will rename `New` to `NewFrom` and `Copy` to `CopyTo`. And that will be the final one.